js/ui/components/clubhouse.js

```diff
@@ -219,6 +279,17 @@ var ClubhouseNotificationBanner = new Lang.Class({
         this.parent();
     },
 
+    // Override this method because we don't want the button
+    // horizontally expanded:
+    addAction: function(label, callback) {
+        let button = new St.Button({ style_class: 'notification-button',
+                                     label: label,
+                                     x_expand: false,
+                                     can_focus: true });
+
+        return this.addButton(button, callback);
```

Comment:
You could just override the `addButton` instead and change the needed properties there (then call the parent's implementation).

Reply:
Good point!